app/controllers/application_controller.rb

```diff
@@ -42,10 +41,4 @@ def clean_date_params(params)
     end
   end
 
-  # if the VEC_MAINTENANCE_UNDERWAY variable is present, redirect to a maintenance page
-  def check_maintenance_page
```

Comment:
Is there a specific reason for removing this?  It seems like it could be really useful for putting up a maintenance page if we ever need downtime?